packages/create-emotion/types/index.d.ts

```diff
@@ -0,0 +1,85 @@
+// Definitions by: Junyoung Clare Jang <https://github.com/Ailrun>
+// TypeScript Version: 2.3
+
+import * as CSS from 'csstype';
+
+export interface MultiDimensionalArray<T> extends Array<T | MultiDimensionalArray<T>> {}
+
+export type CSSBaseObject = CSS.PropertiesFallback<number | string>;
+export type CSSPseudoObject<C> = { [K in CSS.Pseudos]?: CSSObject<C> };
+export interface CSSOthersObject<C> {
+  [propertiesName: string]: Interpolation<C>;
+}
+export interface CSSObject<C> extends CSSBaseObject, CSSPseudoObject<C>, CSSOthersObject<C> {}
+
+export type FunctionInterpolation<C> = (mergedProps: any, context: C) => Interpolation<C> | void;
```

Comment:
Could you remove this? We recently deprecated it in #643 so it would be nice to not have it in the typings.